index.html

```diff
@@ -56,6 +65,9 @@
   </nav>
 </div>
 <div id="particles-js"></div>
+<div id="pos"><h4 id="lg">COPS</h4>
```

Comment:
Don't use ID here, since we may show logo more than once maybe in future in the same page.

Use classes with better names, so it uniquely represents only the logo on the page. **pos** or **lgo** can be repeated unintentionally.
The convention I use for class names are **{section name}-{element usecase}**
For example, I would have probably chosen **header-logo-title** and **header-logo-subtitle**